src/features/amUI/userRightsPage/UserRightsPage.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { useDocumentTitle } from '@/resources/hooks/useDocumentTitle';
+import { useTranslation } from 'react-i18next';
+
+import { Heading } from '@digdir/designsystemet-react';
+
+import { FakePageWrapper } from '../common/FakePageWrapper';
+import { useNavigate, useParams } from 'react-router-dom';
+import { UserIcon } from '@/components/UserIcon/UserIcon';
+import classes from './UserRightsPage.module.css';
+import { PageContainer } from '../common/PageContainer/PageContainer';
+import { PageWrapper } from '@/components';
+import { useGetUserByUUIDQuery } from '@/rtk/features/lookup/lookupApi';
+import { useGetReporteeQuery } from '@/rtk/features/userInfo/userInfoApi';
+import { amUIPath } from '@/routes/paths';
+
+export const UserRightsPage = () => {
+  const { t } = useTranslation();
+  const { id } = useParams();
+
+  const navigate = useNavigate();
+
+  const { data: reportee } = useGetReporteeQuery();
+  const { data: user } = useGetUserByUUIDQuery(id ?? '');
```

Comment:
Should we maybe use getPartyByUUIDQuery instead? Only people have userUuid and user profiles (which getUser fetches) but both people and orgs have partyUuid (userUuid = partyUuid for people) and that is the one we have stored in the browser now anyway. Besides, it saves us one abstraction-level for getting the name and other data 😊

Reply:
Absolutly!